src/ConfigR/BasicConfig.Dictionary.cs

```diff
@@ -57,7 +59,7 @@ public bool ContainsKey(string key)
 
         public bool Remove(string key)
         {
-            log.TraceFormat(CultureInfo.InvariantCulture, "Removing '{0}' from {1}", key, this.GetSource());
+            log.TraceFormat(string.Format(CultureInfo.InvariantCulture, "Removing '{0}' from {1}", key, this.GetSource()));

```

Comment:
Ah, I know why you did this. LibLog doesn't accept an `IFormatProvider`. I think you can just drop it, since it uses `CultureInfo.InvariantCulture` internally. That will allow the removal of the call to `string.Format()`.